tests/Fixer/Alias/NoMixedEchoPrintFixerTest.php

```diff
@@ -292,7 +292,7 @@ public function testWrongConfig($wrongConfig, $expectedMessage)
     {
         $this->setExpectedException(
             'PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException',
-            $expectedMessage
+            $expectedMessage // should be reverted, the regex is more explicit and should stay
```

Comment:
I reverted a bunch of changes in the test where the `RegEx` was dropped. I feel these have value as these test more explicit. Loosing up the test is not needed so I would like not to do it.